scripts/exwave_nawips.sh.ecf

```diff
@@ -0,0 +1,167 @@
+echo "               /nwprod/scripts/exnawips.sh.sms"
+echo " March 2020- Modified for GEFSv12.0"
+#  March-2020 Roberto.Padilla@noaa.gov                                   
+#####################################################################
+
+set -xa
+
+cd $DATA
+
+msg="Begin job for $job"
+postmsg "$jlogfile" "$msg"
+
+cpyfil=gds
+garea=dset
+gbtbls=
+maxgrd=4999
+kxky=
+grdarea=
+proj=
+output=T
+pdsext=no
+g2tbls=g2varswmo2.tbl
+NAGRIB=nagrib2
+
+maxtries=15
+fhcnt=$fstart
+while [ $fhcnt -le $FHMAX_WAV ]; do
+  fhr=$(printf "%03d" $fhcnt)
+
+  for grid in $grids;do
+    case $grid in
+      ao_9km)  grdIDin='arctic.9km'
+               grdIDout='gfswaveao9km' ;;
+      at_10m)  grdIDin='atlocn.0p16'
+               grdIDout='gfswaveat10m' ;;
+      ep_10m)  grdIDin='epacif.0p16'
+               grdIDout='gfswaveep10m' ;;
+      wc_10m)  grdIDin='wcoast.0p16'
+               grdIDout='gfswavewc10m' ;;
+      glo_30m) grdIDin='global.0p25'
+               grdIDout='gfswavegl30m' ;;
+      *)       gridIDin= 
+               grdIDout= ;;
+    esac
+    GRIBIN=$COMIN/$RUNwave.$cycle.$grdIDin.f${fhr}.grib2
+    GRIBIN_chk=$GRIBIN.idx
+    if [ "$grdIDin" = "global.0p25" ]; then
+      #cp $GRIBIN  tempgf.$grdID.f${fhr}
+      #cp $GRIBIN tempgf.$grdIDin.f${fhr}
+      $WGRIB2 -lola 0:720:0.5 -90:361:0.5 gribfile.$grdIDout.f${fhr}  grib \
+                                          $GRIBIN 1> out 2>&1
+      OK=$?
+      if [ "$OK" != '0' ]; then 
+        msg="ABNORMAL EXIT: ERROR IN interpolation the global grid"
+        postmsg "$jlogfile" "$msg"
+        #set +x
+        echo ' '
+        echo '************************************************************* '
+        echo '*** FATAL ERROR : ERROR IN making  gribfile.$grdID.f${fhr}*** '
+        echo '************************************************************* '
+        echo ' '
+        echo $msg
+        #[[ "$LOUD" = YES ]] && set -x
+        echo "$RUNwave $grdID prdgen $date $cycle : error in grbindex." >> $wavelog
+        err=2;export err;err_chk
+      else
+        #cp $GRIBIN gribfile.$grdID.f${fhr}
+        GRIBIN=gribfile.$grdIDout.f${fhr}
+      fi
+    fi
+    echo $GRIBIN  
+
+    GEMGRD=${grdIDout}_${PDY}${cyc}f${fhr}
+    icnt=1
+    while [ $icnt -lt 1000 ]; do
+      if [ -r $GRIBIN_chk ] ; then
+        break
+      else
+        let "icnt=icnt+1"
+        sleep 20
+      fi
+      if [ $icnt -ge $maxtries ]; then
+        msg="ABORTING after 5 minutes of waiting for $GRIBIN."
+        postmsg "$jlogfile" "$msg"
+        echo ' '
+        echo '**************************** '
+        echo '*** ERROR : NO GRIB FILE *** '
+        echo '**************************** '
+        echo ' '
+        echo $msg
+        [[ "$LOUD" = YES ]] && set -x
+        echo "$RUNwave $grdID ${fhr} prdgen $date $cycle : GRIB file missing." >> $wavelog
+        err=1;export err;${errchk} || exit ${err}
+      fi
+    done
+    cp $GRIBIN grib_$grid
+
+    startmsg
+
+    $NAGRIB << EOF
```

Comment:
Use <<- instead of << so you can properly indent the heredoc. (Indentation must be tabs, which is the shell standard anyway).

Reply:
Done.